cmd/osctl/cmd/routes.go

```diff
@@ -41,10 +41,28 @@ var routesCmd = &cobra.Command{
 
 func routesRender(reply *networkapi.RoutesReply) {
 	w := tabwriter.NewWriter(os.Stdout, 0, 0, 3, ' ', 0)
-	fmt.Fprintln(w, "INTERFACE\tDESTINATION\tGATEWAY\tMETRIC")
+	fmt.Fprintln(w, "NODE\tINTERFACE\tDESTINATION\tGATEWAY\tMETRIC")
 
-	for _, r := range reply.Routes {
-		fmt.Fprintf(w, "%s\t%s\t%s\t%d\n", r.Interface, r.Destination, r.Gateway, r.Metric)
+	for _, resp := range reply.Response {
+		var node, dst, gw string
+
+		if resp.Metadata != nil {
+			node = resp.Metadata.Hostname
+		}
+
+		for _, route := range resp.Routes {
+			dst = route.Destination
+			if dst == "<nil>" {
```

Comment:
I feel like we should check if its nil on the server side and set it to `0.0.0.0`.